terraform-environments/hosts-user-data.sh

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/bash
+
+
+# set variables
+  sudo_path=/bin/sudo
+  apt_path=/bin/apt
+  curl_path=/bin/curl
+  sed_path=/bin/sed
+
+# USERNAME=$username
+set -u
+set -e
+set +x
+set -o pipefail
+
+cd /
+
+# install python3.11
+$sudo_path $sed_path -i "s/#\$nrconf{kernelhints} = -1;/\$nrconf{kernelhints} = -1;/g" /etc/needrestart/needrestart.conf
+$sudo_path $sed_path -i 's/#$nrconf{restart} = '"'"'i'"'"';/$nrconf{restart} = '"'"'a'"'"';/g' /etc/needrestart/needrestart.conf
+$sudo_path $apt_path update && $sudo_path $apt_path -y upgrade
+$sudo_path $apt_path install python3.11 -y
+
+$sudo_path $apt_path install ssh -y
```

Comment:
Is this necessary? When do we use the ssh package from the host? Is it needed to be connected to (it might be, you tell me)?

Reply:
idk henrique once said to install ssh, so its been in there ever since

Reply:
i don't think it is needed on my end, i always assumed it was required for the stuff i would be doing with your code

Reply:
the openssh server is, but I assume that is already installed with ubuntu ec2? check